### PR TITLE
Reach & saltern playercount tweak

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
SSReach had a maximum players of 25 whereas even at maximum slots it had 18 jobs including the 4 assistant slots. This map is smaller than Saltern.

Similarly, Saltern has been bumped to 25 slots since it has 25 roles when all round start are filled so made that a reality (not taking into account antags etc too).

This makes the current map vote options:

- Knightship: 0 to 8 max.
- Reach: 8 to 15 max.
- Saltern: 0 to 25 max
- Packed 15 to no max (Max will be added once we have a bigger map in).
- Pillar (PR currently pending) 20 minimum, will replace packed as having no maximum till box and delta get added.

:cl:
- tweak: Changed map Reach from 25 players max to 15.
- tweak: Changed map Saltern from 20 to 25 max players.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Reach should probably also be 0 if it's smaller than Saltern and Saltern has 0, no?

**Comment by Peptide90** [comment]
> Reach should probably also be 0 if it's smaller than Saltern and Saltern has 0, no?

Makes sense although saltern shouldn't be zero and only is for reasons.


But sure I'll change it back to 0.

**Comment by mirrorcult** [comment]
conflicts

**Comment by Peptide90** [comment]
> conflicts

Fixed but yaml linter still cooked

**Comment by metalgearsloth** [comment]
Sounds like the merge got cooked

File: Resources/Prototypes/Maps/game.yml [hunk 1]
```diff
@@ -102,9 +102,9 @@
   nameGenerator:
     !type:NanotrasenNameGenerator
     prefixCreator: '14'
-  mapPath: /Maps/ssreach.yml
-  minPlayers: 8
-  maxPlayers: 25
+  mapPath: Maps/ssreach.yml
```
Comment (metalgearsloth) [inline Resources/Prototypes/Maps/game.yml:105]:
```suggestion
  mapPath: /Maps/ssreach.yml
```

**Pull request merged**